client/src/components/dataset/sidebars/ExtractMultiple.jsx

```diff
@@ -0,0 +1,362 @@
+import * as _ from 'lodash';
```

Comment:
you shouldnt import the whole of lodash because of its size. instead import just the functions you need eg:
`import { merge } from 'lodash';`